api/comments/views.py

```diff
@@ -79,6 +79,7 @@ class CommentDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, Comm
         is_abuse       boolean            has this comment been reported by the current user?
         has_children   boolean            does this comment have replies?
         can_edit       boolean            can the current user edit this comment?
+        new_mentions   list               guids of mentioned users in content of comment
```

Comment:
I am so curious! How are we differentiating mention links from non-mention links? Hmm ...

![gxxkndw8d6](https://cloud.githubusercontent.com/assets/134455/14958942/1db085ea-105b-11e6-9856-c194032e0f27.gif)